packages/proto/src/.gitignore

```diff
@@ -0,0 +1 @@
+/gen/
```

Comment:
can you update the root .gitignore? it also ignores these files already 

```
packages/proto/src/gen/*
```

Reply:
not sure why we need .gitignores scattered everywhere when we already do repo wide ignores in the root .gitignore